disco/types/message.py

```diff
@@ -43,7 +43,7 @@ class Emoji(SlottedModel):
 
     @cached_property
     def custom(self):
-        return self.id is not None
+        return not isinstance(self.id, Unset)
```

Comment:
tbh, I would just make this a falsey check. `id` can't be zero so its safe, and is slightly more sane (given I think ID technically _could_ be `None` if the API decided to return it as that).